C/tests/c4IndexUpdaterTest.cc

```diff
@@ -423,7 +423,7 @@ TEST_CASE_METHOD(LazyVectorAPITest, "IndexUpdater Set Float Array", "[API][.Vect
     REQUIRE(c4indexupdater_finish(updater, ERROR_INFO()));
 
     auto query = REQUIRED(c4query_new2(db, kC4JSONQuery, alloc_slice(json5(R"({
-            WHERE: ['VECTOR_MATCH()', 'words_index', ['$target']],
+            ORDER_BY: ['APPROX_VECTOR_DIST()', ['.vector'], ['$target']],
             WHAT:  [ ['.word'] ],
```

Comment:
Is ['.word'] intentional here, as opposed to ['.vector'] ?

Reply:
Yes. That's the way the query was before, it returns the matching words.